test/specs/modules/Progress/Progress-test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Progress} from 'stardust';
+
+describe('Progress', () => {
+  it('should be able to receive children', () => {
+    render(
+      <Progress>
+        Child
+      </Progress>
+    ).assertText('Child');
+  });
+
+  it('should create a div with the class of bar', () => {
+    render(<Progress />).findClass('bar');
+  });
+
+  // it('should create a div with the class of progress', () => {
+  //   render(<Progress />).firstClass('progress');
+  // });
```

Comment:
I was having some trouble understanding the React-utils. This test was failing because it found two `progress` classes on `<Progress />`. I'd be interested in pairing on it.